src/fn/upload-app-settings.js

```diff
@@ -1,17 +1,29 @@
 const api = require('../lib/api');
-const environment = require('../lib/environment');
 const fs = require('../lib/sync-fs');
+const path = require('path');
+const { info, error } = require('../lib/log');
+const environment = require('../lib/environment');
 
 module.exports = {
   requiresInstance: true,
   execute: () => {
-    const settings = fs.read(`${environment.pathToProject}/app_settings.json`);
+    const settings = fs.read(path.join(environment.pathToProject, 'app_settings.json'));
     return api().updateAppSettings(settings)
       .then(JSON.parse)
-      .then(json => {
-        // As per https://github.com/medic/medic-webapp/issues/3674, this endpoint
-        // will return 200 even when upload fails.
-        if(!json.success) throw new Error(json.error);
+      .then(response => {
+        if (!response.success){
+          throw new Error(response.error);
```

Comment:
the `else` is not required because of the `throw`

Reply:
Removed